s3/README.md

```diff
@@ -1,16 +1,117 @@
-# S3
+# AWS S3 (Simple Storage Service)
 
-## Args
+Create a S3 bucket as an Acorn with a single click or command.
+
+## Usage
+
+From the CLI you can run the following command to create a S3 bucket.
 
+```shell
+acorn run -n s3-bucket ghcr.io/acorn-io/aws/s3:v0.#.#
 ```
 
-Volumes:   <none>
-Secrets:   <none>
-Containers: <none>
-Ports:     <none>
+From an Acornfile you can create the bucket using the S3 acorn too.
+```cue
+services: s3: {
+	image: "ghcr.io/acorn-io/aws/s3:v0.#.#"
+}
+containers: app: {
+    build: context: "./"
+    ports: publish: ["8080/http"]
+    env: {
+        consumes: ["s3"]
```

Comment:
This needs to be outside the `env` block.

Reply:
Oh duh 🤦‍♂️